### PR TITLE
util/rowDecoder: add unit test

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve? 
Add unit test for util/rowDecoder.

### What is changed and how it works?
```go
▶ go test -cover
OK: 1 passed
PASS
coverage: 90.0% of statements
ok      github.com/pingcap/tidb/util/rowDecoder 0.014s
```

### CODE REVIEW

File: util/rowDecoder/decoder_test.go [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+	cols := []*model.ColumnInfo{c1, c2, c3, c4, c5, c6, c7}
+
+	tblInfo := &model.TableInfo{ID: 1, Columns: cols, PKIsHandle: true}
+	tbl := tables.MockTableFromMeta(tblInfo)
+
+	ctx := mock.NewContext()
+	sc := &stmtctx.StatementContext{TimeZone: time.UTC}
+	decodeColsMap := make(map[int64]Column, len(cols))
+	decodeColsMap2 := make(map[int64]Column, len(cols))
+	for i := range tbl.Cols() {
```
Comment (bb7133) [inline util/rowDecoder/decoder_test.go]:
```suggestion
	for _, col := range tbl.Cols() {
```
How about this?

File: util/rowDecoder/decoder_test.go [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+		if i > 0 {
+			c7.Flag |= mysql.UnsignedFlag
+		}
+		bs, err := tablecodec.EncodeRow(sc, row.input, row.cols, nil, nil)
+		c.Assert(err, IsNil)
+		c.Assert(bs, NotNil)
+
+		r, err := de.DecodeAndEvalRowWithMap(ctx, int64(i), bs, time.UTC, timeZoneIn8, nil)
+		c.Assert(err, IsNil)
+		// last column is pk, pk value won't store, and the pk column value is null after decode. we won't check the pk column.
```
Comment (bb7133) [inline util/rowDecoder/decoder_test.go]: I got confused by `pk value won't store`, what does this mean? Thank you!

Comment (crazycs520) [reply util/rowDecoder/decoder_test.go]: refined comment, PTAL.

**Comment by bb7133** [comment]
LGTM, please fix the conflict

**Pull request merged**